### ISSUE TITLE
WordPress, Squarespace, Blogger, and Medium site importer UI fixes

### ISSUE THREAD

**Issue by dezzie** [Automattic/wp-calypso#31092]
This is proposing usability fixes to the current file-based importers. They are related to the more serious ones affecting the URL-based importer here: https://github.com/Automattic/wp-calypso/issues/30564

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/settings/import/freeplandzg.wordpress.com
2. Select `Start Import`

#### What I expected
Consistent and appropriate button components for a wizard-style flow.

#### What happened instead
The action buttons do not stay in a consistent place on the UI. Buttons do not have consistent or appropriate styles. This inhibits user focus. Sometimes it is because buttons are being used as affordances to hide sections of the UI. Other times it is because button components are tied to individual steps instead of treating the full flow as a progression of related actions. 

#### Browser/OS version
All

#### Current Start
![start](https://user-images.githubusercontent.com/1301932/53520699-ddf9fa80-3a9b-11e9-82d1-90ce63beb4a6.png)
#### Proposed
![screen shot 2019-01-25 at 11 42 44-fullpage](https://user-images.githubusercontent.com/1301932/53520714-e94d2600-3a9b-11e9-9b49-99687aeec963.png)
- [ ] `Close` is-scary button moves to the bottom-right, and becomes secondary button
- [ ] Change copy of current `Close` button to `Cancel`
 

#### Current file upload
![2-squarespace-import](https://user-images.githubusercontent.com/1301932/53520729-f23df780-3a9b-11e9-8acb-046c58289245.png)
#### Proposed
![screen shot 2019-01-25 at 11 43 16-fullpage](https://user-images.githubusercontent.com/1301932/53520813-24e7f000-3a9c-11e9-805c-c3bd258fd1eb.png)
- [ ] Maintain disabled state as we do currently

#### Current file validated
![3-squarespace-import](https://user-images.githubusercontent.com/1301932/53520741-fa963280-3a9b-11e9-9d77-b412f723dfab.png)
#### Proposed
![screen shot 2019-01-25 at 11 44 51-fullpage](https://user-images.githubusercontent.com/1301932/53520819-287b7700-3a9c-11e9-90e9-194358f19441.png)
- [ ] Move `Continue` button out of the file upload area and place to the right of the `Cancel` button.

#### Current review import
![4-squarespace-import](https://user-images.githubusercontent.com/1301932/53520759-041f9a80-3a9c-11e9-8e27-22504eb5ae89.png)
#### Proposed
![start-import](https://user-images.githubusercontent.com/1301932/53520832-329d7580-3a9c-11e9-8d32-ce2aef4b26a7.png)
- [ ] Maintain the secondary button from the previous step
- [ ] Change the copy of secondary button to `Back` (<--nice to have not as big a deal)

#### Current import progress
![5-squarespace-import](https://user-images.githubusercontent.com/1301932/53520778-0d106c00-3a9c-11e9-8824-ab493d180967.png)
#### Proposed
![import in progress](https://user-images.githubusercontent.com/1301932/53520839-37622980-3a9c-11e9-976b-fde58b4ba348.png)
- [ ] Use an in-progress button state for the primary button
- [ ] Maintain the current `Importing...` copy for that button in this step
- [ ] Remove the secondary `Back` button for the remainder of the flow

#### Current Done
![7-squarespace-import](https://user-images.githubusercontent.com/1301932/53520790-1568a700-3a9c-11e9-9454-0233b5714c97.png)
#### Proposed
![import complete](https://user-images.githubusercontent.com/1301932/53520845-3d580a80-3a9c-11e9-962f-e6a3f853cec9.png)
- [ ] Maintain the new bottom-right alignment from the previous steps for the `Done` button

**Comment by andfinally** [comment]
I'm not sure if this one could be addressed at the same time, but I'm finding the progress bar in the import file upload pane looks a bit too wide recently:

![image](https://user-images.githubusercontent.com/1647564/53879355-2a968600-4006-11e9-9b17-c8750515a52c.png)

**Comment by getdave** [comment]
Before starting here please refer to https://github.com/Automattic/wp-calypso/issues/30564

**Comment by jonathansadowski** [comment]
I pulled myself off the issue for now. If I end up helping with the work ya'll have done again so far, I'll jump back in.

**Comment by creativecoder** [comment]
Fixed in #31364. Thank you @spen!
